README.md

```diff
@@ -34,24 +33,47 @@ For available parameters or help run:
 nextflow run nf-core/rnafusion --help
 ```
 
+## Quick Start
```

Comment:
I think we can remove/rewrite part of this section

Reply:
I've udpated the README, can you check it out? :)